Specification/BrAPI-Schema/BrAPI-Common/Variable.json

```diff
@@ -4,7 +4,15 @@
             "properties": {
                 "additionalInfo": {
                     "description": "A free space containing any additional information related to a particular object. A data source may provide any JSON object, unrestricted by the BrAPI specification.",
-                    "$ref": "../BrAPI-Common/AdditionalInfo.json#/$defs/AdditionalInfo"
```

Comment:
This changes the model, from a single to an array of AdditionalInfo, can this be put in v2.2? See in same changes in other entities.